docs/sdk/serverless-signing/quick-start.md

```diff
@@ -7,557 +7,188 @@ import TabItem from '@theme/TabItem';
 
 ## Start Here
 
-Blockchains like Ethereum have smart contracts that let developers encode logic to change that state. As a key management network, Lit provides a method that allows developers to encode logic that dictates signing.
+This guide will walk you through setting up a connection to the Lit network, defining a Lit Action, and executing the Lit Action. The code snippets can be compiled and run to successfully execute the Lit Action, as long as an Ethereum wallet is initialized using the `ETHEREUM_PRIVATE_KEY` environment variable.
 
-Lit Actions are JavaScript functions that can be used read and write data across blockchains, web2 platforms, and the rest of the web3 world. You can use Lit Actions to generate signatures when your specified on or off-chain conditions are met, fetch data from off-chain platforms, and manage permissions for PKPs.
+If you're instead looking to learn more about the concept of Lit Actions, check out our [Overview](./overview).
```

Comment:
> If you're instead looking to learn more about the concept of Lit Actions, check out our [Overview](./overview).

Consider "If you're instead looking to learn more about how Lit Actions work, check out the [Overview](./overview) page.